pwa-devdocs/src/tutorials/cloud-deploy/index.md

```diff
@@ -235,7 +177,13 @@ hooks:
         [ -s "$NVM_DIR/nvm.sh"  ] && \. "$NVM_DIR/nvm.sh"
         nvm install --lts=dubnium
 
-        yarn install
+        npx @magento/pwa-buildpack create-project /tmp/pwa --template venia-concept --backend-url https://[your-cloud-url-here] --name Venia --author Magento --install false --npm-client yarn
```

Comment:
Does this mean developers will have to set up their project so that it can be deployed using the `create-project` tool? Which would mean that they will need to create their own [`create.js`](https://magento.github.io/pwa-studio/pwa-buildpack/scaffolding/custom-project-templates/) instruction file for the command?

Reply:
I would consider these instructions for people wanting to use `venia-concept` as a turn key solution; but this was a reason I wanted @zetlen to take a peek at these instructions.

It looked like `create-project` is capable of accepting any npm package as a template, but now requires you select one from a list (`venia-concept` being the only option). If we removed that list restriction, anyone who started with a `venia-concept` scaffolded app could publish and use `--template @awesome-si/super-cool-pwa`.

This falls in line with that production deployment story where we've positioned ourselves as unopinionated. We give you instructions for how to produce your bundle, mostly in a local context, but don't get into too much detail on how to make it public.

Reply:
The `create-project` template functionality is not officially supported yet; there are some kinks to work out, and I haven't notated them yet because I jumped hurriedly off the project to work on BuildBus. So I wouldn't document that quite yet.

Reply:
Yeah, it looked like the logic was built out far enough to support `create-pwa`, but I could see the long term intention. That should be fine for now, with the title note that local builds are suitable if you're not down for turn key `venia-concept`.